images/cross/Makefile

```diff
@@ -20,6 +20,12 @@ CACHE_IMAGES = $(IMAGES)
 
 BASEIMAGE = $(BUILD_REGISTRY)/base-$(GOARCH)
 
+if [[ "$(getconf _NPROCESSORS_ONLN)" == "1" ]]; then
+  makeargs=-j1
+else
+  makeargs=-j$(( $(getconf _NPROCESSORS_ONLN) - 1 ))
```

Comment:
I'd suggest to remove the -1 instead. I don't see the need of it here.
So adding some simple but longer code around to workaround the -1 seems too much for me.
The -1 doesn't protect against any overload since we are more struggled by IOs than Cpu.

Reply:
done, thanks!